readme.rst

```diff
@@ -229,6 +229,32 @@ your doc files were placed (repository root or the ``docs`` directory):
    make install
    make html
 
+Build a PDF
+^^^^^^^^^^^
+
+The documentation can build a PDF locally with the following command:
+
+.. code-block:: none
+
+    make pdf
+
+PDF generation requires some system files. If these files are not found, a prompt will be presented and the generation will stop.
+
+Required packages can be installed with:
+
+.. code-block:: none
+
+    make pdf-prep-force
+
+.. note::
+    
+    When generating a PDF, the index page is considered a 'foreword' and will not be labelled with a chapter.
+
+.. important::
+    
+    When generating a PDF, it is important to not use additional headings before the ``toctree`` on the index page. Documents referenced by the
+    ``toctree`` will be nested under any provided headings. 
```

Comment:
Maybe explain how to use the class to prevent this?

Reply:
This is referenced in changes to the index.rst file and the doc-cheat-sheet.rst

Reply:
Well, yes, I'll find it if I know to search for `rubric`. If I don't, I'll just assume I can never do this ...